todo_app/app.py

```diff
@@ -1,32 +1,50 @@
+from todo_app.item import TrelloBoard
+from todo_app.trello_api_client import TrelloAPIClient
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item, delete_item, get_item, save_item
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
+trello_api_client = TrelloAPIClient()
+boards = trello_api_client.get_boards()
+
 @app.route('/')
 def index():
-    todo_items = sorted(get_items(), key=lambda items: items['status'], reverse = True)
-    return render_template('index.html', todo_items = todo_items)
-
-@app.route('/', methods=['POST'])
-def add_todo_item():
-    title = request.form.get('add-item')
-    add_item(title)
-    return redirect('/')
-
-@app.route('/complete/<id>', methods=['POST'])
-def complete(id):
-    item = get_item(id)
-    item['status'] = 'Completed'
-    save_item(item)
-    return redirect('/')
-
-@app.route('/delete/<id>', methods=['POST'])
-def delete(id):
-    delete_item(id)
-    return redirect('/')
+    selected_board = boards[0]
+    selected_board_items = trello_api_client.get_items_on_a_board(selected_board.id)
+    selected_board_lists = trello_api_client.get_lists_on_a_board(selected_board.id)
+    return render_template('index.html', selected_board = selected_board, selected_board_lists = selected_board_lists, selected_board_items = selected_board_items, boards = boards)
+
+@app.route('/select_board/<id>', methods=['GET'])
```

Comment:
This is a cool way of managing your todo app, and allowing multiple boards rather than fixing just one.
This method has a lot of overlap with the index - one alternative way to break this up would be to store the choice of board id in the users session (as we did with their items previously) - this way index would take care of the effect of this method anyway (by checking if a particular board was chosen, and defaulting to the 0th board if not)